01-leap/leap_spec.rb

```diff
@@ -0,0 +1,48 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/pride'
+require 'minitest/autorun'
+require_relative 'leap'
+
+class Date
+  def leap?
+    throw "Implement this yourself instead of using Ruby's implementation."
+  end
+
+  alias_method :gregorian_leap?, :leap?
+  alias_method :julian_leap?, :leap?
+end
+
+describe "Leap Year Test" do
+  it "Test on 1996" do
+    expect(leap_year?(1996)).must_equal true
+  end
+
+  it "Checking non leap year" do
+    expect(leap_year?(1997)).wont_equal true
+  end
+
+  it "Check non leap even year" do
+    skip
```

Comment:
Students should work to solve the unskipped tests and then delete the skips and work through the tests 1-at-a-time.